codes/classical/properties/QC-LDPC

```diff
@@ -0,0 +1,24 @@
+
+code_id: QC-LDPC
+
+name: 'Quasi-Cyclic LDPC Codes'
+introduced: '\cite{arXiv:1407.5364}'
+
+
+Description:|
```

Comment:
Is this an entry for quasi-cyclic LDPC codes or quasi-cyclic LDPC codes that are based on lifted protographs? What are the connection between them? Refs. [9-22] seem to imply the former codes have been around much longer.

Reply:
it's based on protographs; meaning instead of building the bipartite graph directly, they build a smaller graph and duplicate it over and over in a certain way.